assets/runtime/functions

```diff
@@ -562,7 +562,10 @@ gitlab_configure_oauth_saml() {
       OAUTH_SAML_ISSUER \
       OAUTH_SAML_NAME_IDENTIFIER_FORMAT \
       OAUTH_SAML_GROUPS_ATTRIBUTE \
-      OAUTH_SAML_EXTERNAL_GROUPS
```

Comment:
Logical way for an optional way of `OAUTH_SAML_PRIVATE_KEY` and `OAUTH_SAML_CERTIFICATE` .